### DISCUSSION TITLE
Zoom Bunch

### DISCUSSION THREAD

**Discussion by mikeerickson** [ttscoff/bunch-docs#155] [Q&A] [answered]
I am creating `Zoom` bunch and one of the things I would like to do is turn on Do Not Disturb (using Big Sur)

I have scoured the web for various AppleScripts to do this, but none of them are working.

It does not nave to be AppleScript, just need a way to control it from Bunch, so anything that Bunch can trigger that works, that is fine by me.

**Comment by ttscoff** [comment, marked as answer]
The do not disturb command should work fine on Big Sur:

(dnd on)

-Brett

**Comment by mikeerickson** [comment]
Wow, that was too easy!  Where in the heck did I miss this in the documentation? Is this in there anywhere? I am sure I missed it, you are the documentation king now Mr Technical Writer

Thanks Man!

**Comment by mikeerickson** [comment]
I knew it would be there, just found it

https://bunchapp.co/docs/bunch-files/commands//

**Comment by ttscoff** [comment]
It's a lot of docs to pore through :).

-Brett
